core/tests/test_apps.nem2.hash_lock.py

```diff
@@ -0,0 +1,36 @@
+from common import *
+
+if not utils.BITCOIN_ONLY:
+    from apps.nem2.helpers import *
+    from apps.nem2.hash_lock.serialize import serialize_hash_lock
+    from trezor.messages.NEM2TransactionCommon import NEM2TransactionCommon
+    from trezor.messages.NEM2HashLockTransaction import NEM2HashLockTransaction
+    from trezor.messages.NEM2Mosaic import NEM2Mosaic
+
+@unittest.skipUnless(not utils.BITCOIN_ONLY, "altcoin")
+class TestNem2HashLock(unittest.TestCase):
+    # AggregateBonded
+    # This test uses a TransferTransaction and a MosaicDefinition transaction as part of the AggregateTransaction
+    def test_serialize_hash_lock(self):
+        common = NEM2TransactionCommon(
+            network_type=NEM2_NETWORK_MIJIN_TEST,
+            type=16712,
+            version=36865,
+            max_fee=0,
+            deadline=113728610090
+        )
+
+        hash_lock_transaction = NEM2HashLockTransaction(
+            mosaic=NEM2Mosaic(
+                id="85BBEA6CC462B244",
+                amount=10000000
+            ),
+            duration=480,
+            hash="4FF8C7E00B02D7380900B7795B4CDDC5B9D03536B5BD896BF54F8E5C53F4A32B"
```

Comment:
oh so is the hash of an AggregateBonded transaction containing a transfer and a MosaicDefinition?